fearless/Common/Helpers/ChainAssetsFetching.swift

```diff
@@ -0,0 +1,388 @@
+import RobinHood
+import Foundation
+import FearlessUtils
+
+protocol ChainAssetFetchingProtocol {
+    func fetch(
+        filters: [ChainAssetsFetching.Filter],
+        sortDescriptors: [ChainAssetsFetching.SortDescriptor],
+        completionBlock: @escaping (Result<[ChainAsset], Error>?) -> Void
+    )
+}
+
+final class ChainAssetsFetching: ChainAssetFetchingProtocol {
```

Comment:
please make all functions that not used from outside private

Reply:
all methods except protocol one are placed in private extension, doesn't it enough?